### PR TITLE
Schema change for user feedback feature

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

**Comment by gentlecat** [comment]
I'm going to merge this since I'm short on time and need to start testing feedback stuff on beta very soon. There are no changes to existing tables, this just adds another one.

But please take a look afterwards.

File: admin/sql/create_tables.sql [hunk 1]
```diff
@@ -147,4 +147,11 @@ CREATE TABLE api_key (
   created   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
+CREATE TABLE feedback (
+  user_id            INTEGER, -- PK, FK to user
+  highlevel_model_id INTEGER, -- PK, FK to highlevel_model
+  correct            BOOLEAN NOT NULL,
```
Comment (alastair) [inline admin/sql/create_tables.sql]: This is OK for a 2-class problem, but what if it has more? Do we want a user to be able to select a specific correct answer?

File: admin/sql/create_tables.sql [hunk 2]
```diff
@@ -147,4 +147,11 @@ CREATE TABLE api_key (
...
+  suggestion         TEXT
```
Comment (alastair) [inline admin/sql/create_tables.sql]: Is this freeform?

File: admin/sql/create_tables.sql [hunk 3]
```diff
@@ -147,4 +147,11 @@ CREATE TABLE api_key (
   created   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
+CREATE TABLE feedback (
```
Comment (alastair) [inline admin/sql/create_tables.sql]: can we make the table name more specific? `highlevel_feedback` ?

File: admin/sql/create_tables.sql (hunk 1 above)
Comment (gentlecat) [inline admin/sql/create_tables.sql]: This one is to know if the output is correct or not. Not to specify an expected result.

File: admin/sql/create_tables.sql (hunk 2 above)
Comment (gentlecat) [inline admin/sql/create_tables.sql]: No, name of the expected class is supposed to go there. Perhaps I can rename it to `expected_class` to be more specific.

File: admin/sql/create_tables.sql (hunk 3 above)
Comment (gentlecat) [inline admin/sql/create_tables.sql]: Sounds good.

File: admin/sql/create_tables.sql (hunk 1 above)
Comment (alastair) [inline admin/sql/create_tables.sql]: 👍

File: admin/sql/create_tables.sql (hunk 2 above)
Comment (alastair) [inline admin/sql/create_tables.sql]: 👍

File: admin/sql/create_tables.sql (hunk 3 above)
Comment (alastair) [inline admin/sql/create_tables.sql]: 👍

**Pull request merged**